gulpfile.js

```diff
@@ -20,7 +22,9 @@ require('./').gulpDoxx(require('gulp'),doxxOptions);
 
 
 var ghPagesOptions = {
-
+  config:{
+    remoteUrl: 'git@github.com:pattern-library/pattern-library-utilities.git'
+  },
   src: ['./docs/**/*'],
```

Comment:
if you run `gulp doxx`, it creates a documentation site of all the code in this repository.

that `gulp doxx` task is set up on line 19:
` require('./').gulpDoxx(require('gulp'),doxxOptions);`
which gets the task from lib/gulp-tasks/doxx.js

the configuration is the `doxxOptions` object above line 19

Sorry for the confusion, I neglected to write documentation in this file.